src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -1,55 +1,57 @@
 package com.timelogr.enterprise;
 
+import com.timelogr.enterprise.service.ITimeLogrService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+// ALL WEB FILES GO IN THE resources/templates DIRECTORY
 @Controller
 @RequestMapping("/")
 public class TimeLogrController {
-    private TimeLogrServices timeLogrServices;
 
+    private final ITimeLogrService timeLogrService;
 
     @Autowired
-    public TimeLogrController(TimeLogrServices timeLogrServices){
+    public TimeLogrController(ITimeLogrService timeLogrService){
         super();
-        this.timeLogrServices = timeLogrServices;
+        this.timeLogrService = timeLogrService;
     }
 
-
-
+    /**
+     * Starts Program in the resources/templates/index.html
+     */
     @RequestMapping("/")
     public String index(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
-        return "index"; //STARTS PROGRAM IN THE resources/templates/index.html
+        model.addAttribute("employee", this.timeLogrService.getAllEmployees());
+        return "index";
     }
 
     @RequestMapping("/clients")
-    public String Clients(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
+    public String clients(Model model){
+        model.addAttribute("employee", this.timeLogrService.getAllEmployees());
```

Comment:
Changed method/function names to follow proper Java naming conventions (more info found here): [https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html). I also changed the timeLogrServices.GetAllEmployees() to one, follow naming conventions and for two, to link with the ITimeLogrInterface

Reply:
Definitely... good idea.